lib/store.go

```diff
@@ -248,6 +249,27 @@ func (s *store) GetSession(name string) (*Session, string, error) {
 	return session, "", nil
 }
 
+func (s *store) CreateSession(name string) (*Session, string, error) {
```

Comment:
Please sort this above the `GetSession()` method above (in the same order as the interface methods).